pdc_client/runner.py

```diff
@@ -49,10 +66,22 @@ def __init__(self):
         self.logger = logging.getLogger('pdc')
 
     def load_plugins(self):
+        ns, _ = self.parser.parse_known_args()
+        server = ns.__getattribute__('server')
+        if not server:
+            raise TypeError('Server must be specified')
+        config = pdc_client.read_config_file(server)
+        if config:
+            plugins = config[CONFIG_PLUGINS_KEY_NAME]
+            if not plugins:
+                plugins = DEFAULT_PLUGINS
+        else:
+            plugins = DEFAULT_PLUGINS
+
```

Comment:
Seems from line 74 it could be:
```
plugins = DEFAULT_PLUGINS
if config and config[CONFIG_PLUGINS_KEY_NAME]:
     plugins = config[CONFIG_PLUGINS_KEY_NAME]
```

to make it a little shorter.